CMakeLists.txt

```diff
@@ -183,7 +183,7 @@ if(UNIX)
 
   set(CMAKE_MACOSX_RPATH ON)
 
-  add_executable(blobfuse ${BLOBFUSE_HEADER} ${BLOBFUSE_SOURCE} ${AZURE_STORAGE_HEADER} ${AZURE_STORAGE_SOURCE})
+  add_executable(blobfuse ${BLOBFUSE_HEADER} ${BLOBFUSE_SOURCE} ${AZURE_STORAGE_HEADER} ${AZURE_STORAGE_SOURCE} blobfuse/include/permissions.h blobfuse/src/permissions.cpp azure-storage-cpp-lite/adls/include/get_dfs_properties_request.h azure-storage-cpp-lite/adls/src/get_dfs_properties_request.cpp)
```

Comment:
Move blobfuse/include/permissions.h under BLOBFUSE_HEADER (line 141)
Move blobfuse/include/permissions.cpp under BLOBFUSE_SOURCE (line 152)
Move azure-storage-cpp-lite/adls/include/get_dfs_properties_request.h under AZURE_STORAGE_HEADER (line 2)
Move azure-storage-cpp-lite/adls/src/get_dfs_properties_request.cpp to AZURE_STORAGE_CPP (line 90)